### PR TITLE
add config max ram option

### LINKED ISSUE #837: Memory limit in config should have auto (max) setting and should not take up full width

### PR DESCRIPTION
**Pull request by na3d**
Closes #837

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2017-01-12 at 10 27 16 am

File: src/linodes/linode/settings/layouts/EditConfigPage.js [hunk 1]
```diff
@@ -173,7 +173,8 @@ export class EditConfigPage extends Component {
       runLevel: 'default',
       comments: '',
       label: '',
-      ramLimit: 2048, // TODO: fetch from plan
+      ramLimit: 0, // TODO: fetch from plan
```
Comment (mparke) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: do we have an issue to address this TODO

Comment (eatonphil) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: Honestly we should just start by making a task to make tasks out of all TODOs :/. There are a bunch.

Comment (na3d) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: Agreed, it is time to start cleaning up things.  I think a good number of the TODOs and commented out code is already taken care of.

**Pull request merged**